erpnext/crm/report/opportunity_summary_by_sales_stage/opportunity_summary_by_sales_stage.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _
+from six import iteritems
+import json
+
+
+def execute(filters=None):
+	return OpportunitySummaryBySalesStage(filters).run()
+
+class OpportunitySummaryBySalesStage(object):
+
+	def __init__(self,filters=None):
+		self.filters = frappe._dict(filters or {})
+
+	def run(self):
+		self.get_columns()
+		self.get_data()
+		self.get_chart_data()
+
+		return self.columns,self.data,None,self.chart
```

Comment:
```suggestion
		return self.columns, self.data, None, self.chart
```
Fix these formatting issues everywhere